tensorflow/python/keras/benchmarks/keras_io_examples_benchmark/BUILD

```diff
@@ -0,0 +1,61 @@
+# Description:
```

Comment:
Thanks! A few high-level comments:
1. Is the folder name "keras_examples_benchmarks" better?
2. No need of new BUILD and init files in the new folder. Let's keep the folder hold the models and benchmark tests only.
3. `benchmark_util` can be in the main folder, as it's also used by keras_cpu_benchmark, and might be extended with more utils later.

Will review the code details after the above changes are made.

Reply:
1. But I think keras_io_xxx should be more straightforward to other. 
2. So, only I need to run local and we don't need to build the targets for these benchmark in BUILD file? is that correct? 
3.  Okay, I will move it back.